providers/common.rb

```diff
@@ -0,0 +1,83 @@
+action :install do
+  # Use the git recipe to install git
+  run_context.include_recipe 'git'
+  run_context.include_recipe 'python::pip'
+  run_context.include_recipe 'python::virtualenv'
+
+  if new_resource.create_user
+    group new_resource.group do
+      action :create
+    end
+    user new_resource.owner do
+      action :create
+      gid new_resource.group
+    end
+  end
+
+  # If the new_resource.path is nil set the install path to
+  # `/opt/name_attribute`
+  if new_resource.path.nil?
+    path = "/opt/#{ new_resource.name }"
+  else
+    path = new_resource.path
+  end
+
+  # If the config_destination is nil, as by default, default to
+  # /opt/<path>/settings.py
+  if new_resource.config_destination.nil?
```

Comment:
So if I use or it goes over 80 characters and rubocop whines at me and recommends using an if statement. Is there a better way of doing it which doesn't go over 80 characters?